salt/tests/unit/modules/test_metalk8s_service_configuration.py

```diff
@@ -0,0 +1,86 @@
+import os.path
+import yaml
+
+from parameterized import param, parameterized
+
+from salt.exceptions import CommandExecutionError
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+
+import metalk8s_service_configuration
+
+YAML_TESTS_FILE = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)),
+    "files", "test_metalk8s_service_configuration.yaml"
+)
+with open(YAML_TESTS_FILE) as fd:
+    YAML_TESTS_CASES = yaml.safe_load(fd)
+
+
+class Metalk8sServiceConfigurationTestCase(TestCase, LoaderModuleMockMixin):
+    """
+    TestCase for `metalk8s_service_configuration` module
+    """
+    loader_module = metalk8s_service_configuration
+
+    def test_virtual(self):
+        """
+        Tests the return of `__virtual__` function
+        """
+        self.assertEqual(
+            metalk8s_service_configuration.__virtual__(),
+            'metalk8s_service_configuration'
+        )
+
+    @parameterized.expand([
+        param.explicit(kwargs=test_case)
+        for test_case in YAML_TESTS_CASES["get_service_config"]
+    ])
+    def test_get_service_conf(
+        self,
+        configmap_name,
+        result,
+        configmap_obj={},
+        default_csc={},
```

Comment:
Do not put dict as a default value for args, it's better to have None and then do an if to set the value to an empty dict